modules/project-factory/outputs.tf

```diff
@@ -107,6 +107,7 @@ output "project_numbers" {
 output "projects" {
   description = "Project attributes."
   value       = local.outputs_projects
+  sensitive   = true
```

Comment:
this will have wide ranging implications for all callers, let me actually run a few tests: I have not hit this during regular usage, and if we have to mark this sensitive we still need to provide as much information as possible in an additional non-sensitive output